kore/src/Kore/Builtin/Set/Set.hs

```diff
@@ -187,4 +190,5 @@ asTermLike builtin =
     element
         :: (TermLike variable, SetValue (TermLike variable))
         -> TermLike variable
-    element (key, SetValue) = mkApplySymbol elementSymbol [key]
+    element (key, SetValue) =
+        mkApplySymbol (fromElement elementSymbol) [key]
```

Comment:
Partial functions should be avoided. Can this be refactored?

Reply:
see discussion above